patterns/alz/scripts/Start-AMBACleanup.ps1

```diff
@@ -135,7 +135,7 @@ ForEach ($identity in $policyAssignmentIdentities) {
 
     ForEach ($roleAssignment in $identityRoleAssignments) {
 
-        If ($roleAssignment.Description -like '*_deployed_by_amba*') {
+        If ($roleAssignment.Description -eq '_deployed_by_amba') {
```

Comment:
I see "_deployed_by_amba" was added in the metadata of the role assignment, however here the if statement the evaluations is done on ".Description" can you validate whether that is correct?

Reply:
As far as the script line goes, I made the change to look for exact value (case insensitive). Wildcard in this case can lead to something different added by the customer ... very unlikely but possible.

Here they are the screenshots:
- Successful deployment with a pseudoMG name which is more than 10 chars 
![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/37c3664b-15fa-4cd6-a397-c37f6b6b49f7)

- Description for role assignment correctly populated
![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/74d21489-dceb-4acf-8905-33f0e9a2bdb9)

- Successful cleanup
![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/c468d69b-2981-4bd4-bb51-e29f2b06095d)

![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/96b9c1ca-f245-4586-989b-adaa668c075c)

![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/d453a60a-2515-4f64-9e80-e285421f5049)

- Successful deployment
![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/8c514072-1135-4e2c-94c0-f9d4c0f4d1fb)

![image](https://github.com/Azure/azure-monitor-baseline-alerts/assets/26008435/9f299085-eee9-4ea5-b7b2-640f05e5d239)